README.md

```diff
@@ -63,6 +64,13 @@ $ todo clear
 ```
 ![image](https://user-images.githubusercontent.com/29832401/42281475-41f90abc-7fc1-11e8-9222-7600690b86b6.png)
 
+* `lolcat` : Displays the to-do list items as lolcat.
+
+```bash
+$ todo lolcat
+```
+[X-insert image here]
```

Comment:
You can follow this to add a screenshot: https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d.

Reply:
Oh cool! That's a neat trick